apps/web/cypress/e2e/regression/nested_safes_review.cy.js

```diff
@@ -21,20 +21,20 @@ describe('Nested safes review step tests', () => {
     nsafes.clickOnAddNestedSafeBtn()
   })
 
-  it('Verify review step with Fund new assets', () => {
+  it.only('Verify middle step with Fund new assets in create nestedsafe tx flow', () => {
```

Comment:
Is this intended or a leftover from testing? It will result in only this test being run.